### PR TITLE
v4.0.41 Release

### PR DESCRIPTION
**Pull request by duboisp**
## What does this pull request (PR) do? / Que fait cette demande « pull » (PR)?

wet-boew version 4.0.41 release note

### CODE REVIEW

File: site/pages/docs/versions/dwnld-fr.hbs [hunk 1]
```diff
@@ -402,7 +425,29 @@
...
+				<tr>
+					<th scope="col">Thème</th>
+					<th scope="col">Code source</th>
+					<th scope="col">Fichiers de production</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>Boîte à outils de l'expérience Web (BOEW)</td>
+					<td><a href="https://github.com/wet-boew/wet-boew/archive/v4.0.41.zip">Télécharger<span class="wb-inv"> - Code source de v4.0.40 (DCT) - Thème de la Boîte à outils de l'expérience Web (BOEW)</span></a></td>
```
Comment (GormFrank) [inline site/pages/docs/versions/dwnld-fr.hbs:441]:
```suggestion
					<td><a href="https://github.com/wet-boew/wet-boew/archive/v4.0.41.zip">Télécharger<span class="wb-inv"> - Code source de v4.0.41 (DCT) - Thème de la Boîte à outils de l'expérience Web (BOEW)</span></a></td>
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
One small change and then good to go!

**Review by GormFrank — APPROVED** [review]

**Review by GormFrank — CHANGES_REQUESTED** [review]
Travis-ci doesn't like the migration of the download page to a new page. Please fix the build.

**Review by GormFrank — APPROVED** [review]

**Comment by duboisp** [comment]
This release is needed to meet an organisation priority in GCWeb and we include PR's that was ready for deployment.

**Pull request merged**